nntrainer/graph/network_graph.h

```diff
@@ -172,6 +172,8 @@ class NetworkGraph {
    */
   void setBatchSize(unsigned int batch_size);
 
+  void updateTimeStep(unsigned int max_timestep);
```

Comment:
Needs doxygen tag to explain about this function.

Reply:
Added